new/LIT6946InvocationMary.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6946InvocationMary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Invocation to the Virgin Mary and Jesus Christ for protection</title>
+                <title xml:lang="gez" xml:id="t2">ሰአሊ፡ ለነ፡ ቅድስት፡ ድንግል፡ ማርያም።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">saʾali lana qǝddǝst dǝngǝl Māryām</title>
+                <title xml:lang="en" type="normalized" corresp="#t2">Pray for us, Holy Virgin Mary!</title>
```

Comment:
To me, the exclamation mark seems unusual :)

Reply:
I agree and I will delete it.